src/System.Runtime.Numerics/src/System/Numerics/BigNumber.cs

```diff
@@ -469,10 +469,10 @@ private static unsafe bool NumberToBigInteger(ref BigNumberBuffer number, ref Bi
         }
 
         // This function is consistent with VM\COMNumber.cpp!COMNumber::ParseFormatSpecifier
-        internal static char ParseFormatSpecifier(string format, out int digits)
+        internal static char ParseFormatSpecifier(ReadOnlySpan<char> format, out int digits)
         {
             digits = -1;
-            if (string.IsNullOrEmpty(format))
+            if (format.Length == 0)
```

Comment:
Is there a tangible difference between `format.IsEmpty` and `format.Length == 0`?  (And/or do we have a convention of which one we prefer?)

Reply:
They're identical: the implementation of `IsEmpty` just does `Length == 0`.  I typically just forget that `IsEmpty` exists (and actually requested that we not have it, but c'est la vie).